registry/response/terraform_provider.go

```diff
@@ -50,11 +51,38 @@ type TerraformProviderPlatformLocation struct {
 	DownloadURL         string `json:"download_url"`
 	ShasumsURL          string `json:"shasums_url"`
 	ShasumsSignatureURL string `json:"shasums_signature_url"`
+	Shasum              string `json:"shasum"`
```

Comment:
protocols are also included in the download if needed

```
{"protocols": ["4"], ...}
```

Reply:
Added!